_docs/setup/install-kubernetes.md

```diff
@@ -14,57 +14,57 @@ Quick Start instructions to install and configure Istio in a Kubernetes cluster.
+    ```bash
+    gcloud container clusters get-credentials <cluster-name> --zone <zone> --project <project-name>
+    ```
 
-  ```bash
-  $(bx cs cluster-config <cluster-name>|grep "export KUBECONFIG")
-  ```
-* If you are using [Openshift Origin](https://www.openshift.org) version 3.7 or later, Openshift by default does not allow containers running with uid 0, you can enable this for Istio's service accounts for ingress and egress as follows:
+  * If you are using [IBM Bluemix Container Service](https://www.ibm.com/cloud-computing/bluemix/containers), find out your cluster name, and fetch credentials for kubectl:
 
-  ```bash
- oc adm policy add-scc-to-user anyuid -z istio-ingress-service-account -n istio-system
- oc adm policy add-scc-to-user anyuid -z istio-egress-service-account -n istio-system
-  ```  
+    ```bash
+    $(bx cs cluster-config <cluster-name>|grep "export KUBECONFIG")
+    ```
+  * If you are using [Openshift Origin](https://www.openshift.org) version 3.7 or later, Openshift by default does not allow containers running with uid 0, you can enable this for Istio's service accounts for ingress and egress as follows:
 
-* Install the Kubernetes client [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/), or upgrade to the latest
-  version supported by your cluster.
+    ```bash
+    oc adm policy add-scc-to-user anyuid -z istio-ingress-service-account -n istio-system
+    oc adm policy add-scc-to-user anyuid -z istio-egress-service-account -n istio-system
+    ```  
 
-* Upgrading Istio is not yet supported. If you previously installed Istio on this cluster, please uninstall first by following the
-  uninstalling steps and yaml files matching your current version.
-  For the {{ site.data.istio.version }} release, the [uninstalling]({{home}}/docs/setup/install-kubernetes.html#uninstalling) steps are located at the end of this page.
+* Install or upgrade the Kubernetes client [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/) to match
+  version supported by your cluster (version 1.7 or newer for CRD support).
+
+* If you installed one of our v0.1.x alpha versions before please [uninstall](https://istio.io/v-0.1/docs/tasks/installing-istio.html#uninstalling) it first.
 
 ## Installation steps
 
 Starting with the {{ site.data.istio.version }} release, Istio is installed in its own `istio-system` namespace,
 and can manage micro-services from all other namespaces.
 
-1. Go to the [Istio release](https://github.com/istio/istio/releases) page, to download the installation file corresponding to your OS or run 
+1. Go to the [Istio release](https://github.com/istio/istio/releases) page, to download the installation file corresponding to your OS or run
    ```bash
    curl -L https://git.io/getIstio | sh -
-   ``` 
-   to download and extract the latest release automatically (on MacOS and Ubuntu).
+   ```
+   to download and extract the latest stable release automatically (on MacOS and Ubuntu).
```

Comment:
that's an indirect warning that for now this still downloads 0.1.6 (which is a good thing)

Reply:
If these instructions are versioned based on different releases, its a bit weird for e.g. the 0.2 docs to end up installing 0.1.  We should figure out the right way to describe this, and maybe include a more explicit warning, and describe exactly which version getIstio will return, and a way to use getIstio to install a specific version.